src/modules/main/sidebar/Sidebar.tsx

```diff
@@ -32,16 +34,28 @@ const getActiveLinkIdx = (pathname: string, links: SidebarLink[]) => {
   });
 };
 
-export const Sidebar = ({ basePath }: SidebarProps) => {
+export const Sidebar = ({ basePath, isSidebarOpen, changeSidebarState }: SidebarProps) => {
   const { pathname } = useLocation();
 
   const activeLinkIdx = getActiveLinkIdx(pathname.replace(basePath, ''), sidebarLinks);
 
   return (
     <aside className={csx.sidebar}>
       <div className={csx.sidebarContent}>
-        <figure className={csx.logo}>
-          <Logo />
+        <figure className={csx.logo} >
+          {isSidebarOpen 
+            ? 
+              <div style={{
```

Comment:
Avoid inline styles should be replaced with @include row in scsss

Reply:
Inline styles  have been replaced with styles in the .scss file